File: html/web3/cgi-bin/hello_form.py
#!/usr/bin/python

# Import modules for CGI handling
import cgi

# Create instance of FieldStorage
form = cgi.FieldStorage()

# Get data crom fields
fname = form.getvalue('fname')
lname = form.getvalue('lname')

print 'Content-type:text/html\r\n\r\n'
print '<html>'
print '<head>'
print '<title>Hello GET - CGI Program</title>'
print '</head>'
print '<body>'
print '<h2>Hello %s %s</h2>' % (fname, lname)
print '</body>'
print '</html>'

File: html/web3/cgi-bin/hello_form.php
<html>
<head>
<title>Hello PHP CGI</title>
</head>
<body>
<?php
$first_name = htmlspecialchars($_POST['fname']);
$last_name = htmlspecialchars($_POST['lname']);
echo "<h2>Hello " . $first_name . " " . $last_name . "</h2>";
?>
</body>
</html>
